spec/facades/sensor_facade_spec.rb

```diff
@@ -23,4 +23,14 @@
     expect(sensors).to be_an(Array)
     expect(sensors.all? { |sensor| sensor.class == Sensor }).to be_truthy
   end
+
+  it "can delete a sensor", :vcr do
```

Comment:
Maybe we should switch to webmock? Or it'll actually delete real sensors every 7 days. 

But yeah this also goes back to the same convo we've been having on how FE tests are pretty stubbed.

Reply:
I don't know. This feels like a more real test than the webmock. But they're both flawed. I say we keep them as is, ask Ian about this during project evals and put in a ticket to refactor tests later.